src/V3AssertPre.cpp

```diff
@@ -67,6 +69,71 @@ class AssertPreVisitor final : public VNVisitor {
         m_senip = nullptr;
         m_disablep = nullptr;
     }
+    AstPropClocked* getPropertyExprp(const AstProperty* const propp) {
+        // The only statements possible in AstProperty are AstPropClocked (body)
+        // and AstVar (arguments).
+        AstNode* propExprp = propp->stmtsp();
+        while (VN_IS(propExprp, Var)) propExprp = propExprp->nextp();
```

Comment:
Why not keep these in seperate `op` pointers? And just get them directly instead of iterating.

Reply:
Because this class inherits from AstNodeFTask. I think it is needed, because Property references are parsed as AstFuncRef and linking rules are the same.